config-linux.md

```diff
@@ -37,7 +37,7 @@ The following parameters can be specified to set up namespaces:
     * **`time`** the container will be able to have its own clocks.
 * **`path`** *(string, OPTIONAL)* - namespace file.
     This value MUST be an absolute path in the [runtime mount namespace](glossary.md#runtime-namespace).
-    The runtime MUST place the container process in the namespace associated with that `path`.
+    The runtime MUST let the container process join in the namespace associated with that `path`.
```

Comment:
why this wording change?

Reply:
The old wording was better, "let the container process join" is not an accurate description.